native/src/main/java/io/ballerina/stdlib/graphql/runtime/engine/ArgumentHandler.java

```diff
@@ -63,12 +70,15 @@
 public class ArgumentHandler {
     private final BMap<BString, Object> argumentsMap;
     private final MethodType method;
-
     private final BMap<BString, Object> fileInfo;
     private final BObject context;
     private final BObject field;
+    private final BObject responseGenerator;
+    private final Boolean validation;
```

Comment:
Check other places:

```suggestion
    private final Boolean validate;
```

Reply:
Keep this as it is to be consistent with the config variable name `validation`.